packages/rocketchat-ui/client/lib/chatMessages.js

```diff
@@ -182,6 +182,12 @@ this.ChatMessages = class ChatMessages {
 
 			const msg = input.value;
 			const msgObject = { _id: Random.id(), rid, msg};
+			const lastMessage = ChatMessage.findOne({rid}, { fields: { ts: 1 }, sort: { ts: -1 }});
+			if (msg[0]+msg[1] === '+:') {
```

Comment:
Do not trigger a sound, just send the message, maybe the users is trying to send that as a message